### PR TITLE
Mock Server

### PR DESCRIPTION
**Pull request by dtaylor7**

### CODE REVIEW

File: mocks/routes/rpp/customer.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+const { faker } = require('@faker-js/faker/locale/en_GB')
+
+module.exports = [
+  {
+    id: 'rpp-customer-get-by-id',
```
[the remark this answers is not in the document]
Comment (dtaylor7) [reply mocks/routes/rpp/customer.js:5]: sure, can do.... although... i mean... its a mock ;)

File: mocks/routes/rpp/business.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    url: '/rpp/api/organisation/:orgId',
+    method: ['GET'],
+    variants: [
+      {
+        id: 'default',
+        type: 'json',
+        options: {
+          status: 200,
+          body: {
+            _data: business()
```
[the remark this answers is not in the document]
Comment (dtaylor7) [reply mocks/routes/rpp/business.js:83]: its duplicated, so put in a function so i only have to do once...

[the remark this answers is not in the document]
Comment (csaltmountain) [reply mocks/routes/rpp/business.js]: Ah yeah, hadn't spotted the other place where you'd used it

**Review by csaltmountain — APPROVED** [review]

**Review by hadidefra — APPROVED** [review]
Ready to 🚢

**Review by jack-mccabe-defra — APPROVED** [review]
LGTM

**Pull request merged**